### PR TITLE
Misc edits

### PR DESCRIPTION
**Pull request by JHuraibi**

### CODE REVIEW

File: public/tests/instructions_page.html [hunk 1]
```diff
@@ -54,52 +54,60 @@
...
 				</p>
-
+				
 				<input type="range" id="speed" max="5" min="1" style="width:80%" oninput="num.value = this.value">
 				<output id="num"></output>
 				<div class="btn-group">
 					<button type="button" onclick="setspeed()" class="btn btn-sm btn-outline-secondary">Set Speed</button>
 				</div>
 			</div>
+
```
Comment (JHuraibi) [inline public/tests/instructions_page.html:88]: Need okay from Tesneem here

File: public/tests/AmslerGrid/Amsler.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ var countDown = 15;
 function interval()
 {
     document.getElementById("timer").innerHTML = countDown +" seconds";
```
Comment (JHuraibi) [inline public/tests/AmslerGrid/Amsler.js:5]: Need okay from Ali here

File: public/js/CustomInstructionsInjector.js [hunk 3]
```diff
@@ -18,14 +18,13 @@ function retrieveTestTitle(target_test) {
...
 		default:
-			return "[DEBUG] - RETURNED FROM FUNCTION";
+			return "[DEBUG] - NO INSTRUCTIONS RETURNED FROM FUNCTION";
 
-		// // AMSLER GRID
-		// case "amsler_grid":
-		// 	return "Amsler Grid Instructions";
+		// AMSLER GRID
+		case "amsler_grid":
+			return "You are about to take the Amsler Grid test. [Ali add the instructions for your test here]";
```
Comment (JHuraibi) [inline public/js/CustomInstructionsInjector.js:28]: Ali need your instructions here for your Amsler Grid

File: public/tests/instructions_page.html (hunk 1 above)
Comment (fx0285) [reply public/tests/instructions_page.html:88]: Okay!

File: public/tests/AmslerGrid/Amsler.js (hunk 2 above)
Comment (JHuraibi) [reply public/tests/AmslerGrid/Amsler.js:5]: Reverting for now.

File: public/js/CustomInstructionsInjector.js [hunk 4]
```diff
@@ -18,14 +18,13 @@ function retrieveTestTitle(target_test) {
...
```
Comment (JHuraibi) [inline public/js/CustomInstructionsInjector.js:27]: Will need Ali or one of us to complete this.

**Review by JHuraibi — COMMENTED** [review]
Update items related to Ali's tasks when possible.

File: public/tests/AmslerGrid/Amsler.js (hunk 2 above)
Comment (JHuraibi) [reply public/tests/AmslerGrid/Amsler.js:5]: Reverted in commit 1a07500

**Pull request merged**